lmdeploy/archs.py

```diff
@@ -136,6 +136,8 @@ def check_vl_llm(config: dict) -> bool:
         return True
     elif arch == 'LlavaNextForConditionalGeneration':
         return True
+    elif arch == 'Phi3VForCausalLM':
```

Comment:
Can we refactor `check_vl_llm`? Checking if `arch` in a list is more user-friendly than adding `elif` one by one

Reply:
done